pyrit/prompt_converter/character_replacement_converter.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pyrit.models import PromptDataType
+from pyrit.prompt_converter import PromptConverter, ConverterResult
+
+
+class CharacterReplacementConverter(PromptConverter):
+    """Converts a string by replacing chosen character with a new character of choice
```

Comment:
Change to phrase since it can be more than characters
```suggestion
    """Converts a string by replacing chosen phrase with a new phrase of choice
```